.ci/ogc/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-mkdir /usr/src/qgis/build
-cd /usr/src/qgis/build || exit -1
+mkdir -p /usr/src/qgis/build
+cd /usr/src/qgis/build || exit 1
```

Comment:
I think it has nothing to do with the current PR. Can you remove these changes please?